lib/experimental/interactivity-api/script-loader.php

```diff
@@ -39,7 +45,7 @@ function gutenberg_register_interactivity_scripts( $scripts ) {
  * @return string The modified script tag.
  */
 function gutenberg_interactivity_scripts_add_defer_attribute( $tag, $handle ) {
-	if ( str_starts_with( $handle, 'wp-interactivity-' ) ) {
+	if ( str_starts_with( $handle, 'wp-interactivity-' ) || str_contains( $tag, 'view/interactivity.min.js' ) ) {
```

Comment:
Assuming that we always have the Interactivity script in `view/interactivity.min.js` file, we can temporarily enforce `defer`. Otherwise, the runtime might load after the view script.